#!/usr/bin/env python
# -*- coding: utf-8 -*-

PACKAGE = "method_callback"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("param1", int_t, 0,
        "param1", 50, 0, 100)
gen.add("param2", double_t, 0,
        "param2", 2.5, 0.1, 5.0)
gen.add("param3", bool_t, 0,
        "param3", True)

exit(gen.generate(PACKAGE, "method_callback", "MethodCallback"))
